vector/src/main/java/im/vector/app/features/location/UrlMapProvider.kt

```diff
@@ -16,52 +16,22 @@
 
 package im.vector.app.features.location
 
-import im.vector.app.features.raw.wellknown.getElementWellknown
-import org.matrix.android.sdk.api.extensions.tryOrNull
-import org.matrix.android.sdk.api.raw.RawService
-import org.matrix.android.sdk.api.session.Session
+import im.vector.app.features.raw.wellknown.ElementWellKnown
 import javax.inject.Inject
+import javax.inject.Singleton
 
-class UrlMapProvider @Inject constructor(
-        private val session: Session,
-        private val rawService: RawService,
-        locationSharingConfig: LocationSharingConfig,
-) {
-    private val keyParam = "?key=${locationSharingConfig.mapTilerKey}"
```

Comment:
I wonder if we should also remove the `LocationSharingConfig` class and the `MapTiler` API key.
@onurays What do you think? I am asking you since it appears you have added this key in the project and you may have more context about it.

Reply:
After some tests, it seems a fallback is still necessary on `Matrix.org` for example.

Reply:
I added the fallback implementation to the maptiler API, it is configurable via `LocationSharingConfig`